homeassistant/components/devolo_home_network/button.py

```diff
@@ -56,6 +56,7 @@ class DevoloButtonEntityDescription(
     ),
     START_WPS: DevoloButtonEntityDescription(
         key=START_WPS,
+        translation_key="start_wps",
         icon="mdi:wifi-plus",
```

Comment:
I guess, the icon should be removed then?

Reply:
Yes! Oops